requests/utils.py

```diff
@@ -37,6 +38,59 @@
 DEFAULT_CA_BUNDLE_PATH = certs.where()
 
 
+if os.name == 'nt':
+    # provide a proxy_bypass version on Windows without DNS lookups
+
+    def proxy_bypass_registry(host):
+        try:
+            if is_py3:
+                import winreg
+            else:
+                import _winreg as winreg
+
+        except ImportError:
+            # Std modules, so should be around - but you never know!
```

Comment:
Yeah, let's do that: there's no need to tolerate this `ImportError` I don't think.